### PR TITLE
[POL-282] Provide history items for one policy

### PR DESCRIPTION
**Pull request by pilhuhn**
Signed-off-by: Heiko W. Rupp <hrupp@redhat.com>

```
$ curl -i -Hx-rh-identity:`cat rhid.txt` http://localhost:8080/api/policies/v1.0/policies/8671900e-9d31-47bf-9249-8f45698ede72/history/trigger
HTTP/1.1 200 OK
Content-Length: 1711
Content-Type: application/json

[{"ctime":1590506001593,"hostName":"VM 22","id":"d4039530-4e3c-dead-beef-44de55400c2b"},
  {"ctime":1590506001553,"hostName":"VM 22","id":"d4039530-4e3c-dead-beef-44de55400c2b"},
  {"ctime":1590506001551,"hostName":"VM 22","id":"d4039530-4e3c-dead-beef-44de55400c2b"},
  {"ctime":1590506001551,"hostName":"VM 22","id":"d4039530-4e3c-dead-beef-44de55400c2b"},
 {"ctime":1590506001550,"hostName":"VM 22","id":"d4039530-4e3c-dead-beef-44de55400c2b"}]
```

### CODE REVIEW

**Comment by pilhuhn** [comment]
@josejulio That works already, but I need to add tests

File: src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java [hunk 1]
```diff
@@ -428,6 +432,7 @@ private Msg getEngineExceptionMsg(Exception e) {
   @APIResponse(responseCode = "200", description = "Policy deleted")
   @APIResponse(responseCode = "404", description = "Policy not found")
   @APIResponse(responseCode = "403", description = "Individual permissions missing to complete action")
+  @Parameter(name = "id", description = "UUID of the policy", schema = @Schema(type = SchemaType.STRING))
```
Comment (josejulio) [inline src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java:435]: Could you remove the `schema` from here (and the others params that are already a `@PathParam`), the schema seems to be inferred from the variable type.

In this case, you are changing the type from `Uuid` to `String`.

For ui-frontend, it means this change:

```diff
 export interface UsePostPoliciesValidateNameParams {
     body?: string;
-    id?: schemas.Uuid;
+    id?: string;
 }
```

**Comment by josejulio** [comment]
Tested, is returning the trigger history. Using to move on with the UI.

Screenshot_2020-05-27_14-42-11

File: src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java (hunk 1 above)
Comment (josejulio) [reply src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java:435]: I see that you removed the `@Schema` from the other places, but here, could you do the same here? :)

**Pull request merged**